PC/svm/includes/common.h

```diff
@@ -0,0 +1,73 @@
+#ifndef COMMON
+#define COMMON
+
+#include "json.hpp"
+
+#define DATA_DIMENSIONS 2 // for Hu moments, it will be 7
+
+using json =nlohmann::json;
+
+std::vector<float> svm_build_single_element(json &element, bool debug = false)
+{
+    std::vector<float> training_element;
+    if(debug)
+        std::cout << "element: " << element << std::endl; 
+    training_element.push_back(element["x"]);
+    training_element.push_back(element["y"]);
+    return training_element;
+}
+
+void print_vector(std::vector<float> &vect)
```

Comment:
We will have conflicts, as I already moved it to modules/print_vector.cpp on my branch. But alright, you were first ;)